chapters/methodology/theory.tex

```diff
@@ -144,37 +169,43 @@ \subsubsection{CADIS-$\Omega$}
 particle weights according to their importance. However, CADIS-$\Omega$ will
 use the
 $\Omega$-adjoint scalar flux rather than the standard adjoint scalar flux
-to generate the biased source distribution and the particle weights.
+to generate the biased source distribution, weight windows, and particle birth weights.
 The consistency between the biased source and particle weights is maintained
-when using $\phi_{\Omega}^{\dagger}$ rather than $\phi^{\dagger}$. The adjusted
+when using $\phi_{\Omega}^{\dagger}$ rather than $\phi^{\dagger}$. 
+% I don't understand the above sentence. 
+The adjusted
 formulation of CADIS using the $\Omega$ fluxes is given by Eqs.
-\ref{eq:CADISomegamethod}.
+\eqref{eq:CADISomegamethod}.
+% note: always use eqref--will format equation references more clearly.
 %
 \begin{subequations}
 \label{eq:CADISomegamethod}
 %
-The biased source distribution used by CADIS-$\Omega$ is given by:
+The biased source distribution used by CADIS-$\Omega$ is formulated just as
+it is in CADIS, except the new adjoint fluxes are used:
+% I would refer to all of our quantities with the sub omega to make it clear
 \begin{equation}
 \begin{split}
-  \hat{q}  & = \frac{\phi_{\Omega}^{\dagger}(\vec {r} ,E)q(\vec {r} ,E)}
+  \hat{q}_{\Omega}  & = \frac{\phi_{\Omega}^{\dagger}(\vec {r} ,E)q(\vec {r} ,E)}
                {\iint\phi_{\Omega}^{\dagger}(\vec {r} ,E)
                q(\vec {r} ,E) dE d\vec{r}} \\
-           & = \frac{\phi_{\Omega}^{\dagger}(\vec {r} ,E)q(\vec {r} ,E)}{R}.
+           & = \frac{\phi_{\Omega}^{\dagger}(\vec {r} ,E)q(\vec {r} ,E)}{R_{\Omega}}.
 \end{split}
 \end{equation}
-%
-The  starting weights of the particles sampled from the
-biased source distribution, $\hat{q}$ are given by
+% I think this is correct about R? You probably want to call out that R is now
+% made with the omega flux
```

Comment:
Good call. To clarify, are you asking:
- I think it is correct that R should be calculated with the omega flux as R_omega = phi_omega*q
or are you asking
- I thought R was calculated with the standard phi^dagger*q, and we only use the phi_omega in the numerator.